app/services/amalgamator/processor/base.rb

```diff
@@ -47,7 +47,7 @@ def adjust_flexible_shares(dataset_c_values)
 
       # Rounds values to 8 decimals and replaces nil with 0.0
```

Comment:
If we have changed the method from `round` to `floor`, should'nt the comment be updated as well?